search/geocoder.cpp

```diff
@@ -353,7 +353,7 @@ void Geocoder::SetParams(Params const & params)
         // Here and below, we use LevenshteinDFAs for fuzzy
         // matching. But due to performance reasons, we assume that the
         // first letter is always correct.
```

Comment:
Move the comment to BuildLevenshteinDFA, please.

Reply:
Done